users/updates/components/Tm5xY22MM8eC.php

```diff
@@ -8,7 +8,7 @@
 $countE=0;
 $db->insert('updates',['migration'=>$update]);
 
-$find = $db->query("SELECT id,label FROM menus WHERE label = ?",array("User Management"))->results();
+$find = $db->query("SELECT id,label FROM menus WHERE label = ?", ["User Management"])->results();
```

Comment:
What was wrong with the original syntax?

Reply:
The same reason as mentioned before. I can revert it if you want